mongo/options/mongooptions.go

```diff
@@ -89,16 +89,22 @@ const (
 	After
 )
 
```

Comment:
Please add `changeStreamPreAndPostImages` to `CreateCollectionOptions`. I think this may be a requirement from the GODRIVER-2294:

> The create and collMod commands now support a changeStreamPreAndPostImages option that accepts: {enabled: <boolean>}

Reply:
Excellent catch. As discussed offline, the addition of `changeStreamPreAndPostImages` to create collection options was not really included in the original spec changes for [DRIVERS-1915](https://jira.mongodb.org/browse/DRIVERS-1915). DRIVERS-2294 was filed to add spec tests for drivers' `createCollection` and `collMod` helpers if they exist. I think I'll do that spec work and add `ChangeStreamPreAndPostImages` to `CreateCollectionOptions` with GODRIVER-2396.